reporting.md

```diff
@@ -0,0 +1,126 @@
+# Simeon Cloud Power BI Reporting
+Report on your Simeon Sync data from a single location. Use one of our builtin reports or create your own from the raw data.
+
+## Tenant updates
+When installing Simeon Cloud Power BI Reporting, the installer will make the following changes to the tenant selected to host the Power BI report:
+- Creates an **Azure Service Principal** named Simeon Cloud Power BI Reporting
+- **Generates credentials** for Simeon Cloud Power BI Reporting and saves the credentials in an Azure DevOps Variable Group shared only with your tenant pipelines
```

Comment:
Generates a client secret....and saves it as a secure variables in an Azure DevOps....